virtio-devices/src/block.rs

```diff
@@ -789,9 +789,14 @@ impl Block {
             self.id,
             self.disk_path.display()
         );
+        let granularity = self
+            .disk_image
+            .size()
+            .map(|size| LockGranularity::ByteRange(0, size))
+            .unwrap_or(LockGranularity::WholeFile);
```

Comment:
Just so I understand everything correctly:

This will basically always result in a byte sized lock?

Even if we do not make it configurable for the user right now, would it make sense to pass the lock granularity more explicitly?